pageserver/src/http/models.rs

```diff
@@ -161,3 +161,21 @@ pub struct TimelineInfo {
     pub local: Option<LocalTimelineInfo>,
     pub remote: Option<RemoteTimelineInfo>,
 }
+
+pub type ConfigureFailpointsRequest = Vec<FailpointConfig>;
+
+/// Information for configuring a single fail point
+#[derive(Debug, Serialize, Deserialize)]
+pub struct FailpointConfig<S = String> {
```

Comment:
I've never seen the `<S = String>` syntax before. What does it do? Why not just use String directly?

Reply:
Gotcha, thanks! Just when you think you know the language, you discover a new syntax :-). If anyone else is wondering about this, this explains it: https://doc.rust-lang.org/book/ch19-03-advanced-traits.html#default-generic-type-parameters-and-operator-overloading.